realtime/src/main/java/com/mx/path/service/connection/realtime/models/MdxAccount.java

```diff
@@ -1,4 +1,4 @@
-package com.mx.path.api.lib.realtime.models;
+package com.mx.path.service.connection.realtime.models;
```

Comment:
This should probably be `com.mx.path.service.connection.realtime.model`

Reply:
fixed